clowdapp.yaml

```diff
@@ -295,25 +295,23 @@ objects:
           value: "secret_key_base"
 
 parameters:
-- name: MIN_REPLICAS
```

Comment:
`MIN_REPLICAS` is being removed. Is that correct?

Reply:
yeah that's not used anywhere, it was probably a leftover, we're using per-service number-of-replicas variables, like `REPLICAS_BACKEND`